{{ cookiecutter.__project_name_kebab }}/ruff.toml

```diff
@@ -0,0 +1,30 @@
+extend-exclude = [
+    "migrations",
+    "LC_MESSAGES",
+    "locale",
+]
+line-length = 88
+target-version = "py38"  # minimum target version
+
+
+[lint]
+# E: pycodestyle errors
+# F: Pyflakes
+# W: Warning
+# I: isort
+# B: flake8-bugbear
+# C4: flake8-comprehensions
+select = ["E", "F", "W", "I",  "B", "C4"]
```

Comment:
this may not be _exactly_ the same as what we were running before, but it is pretty close

Reply:
I wonder if we can extend this a bit via `extend-select` and include  

```
"DJ",  # flake8-django
"S",   # flake8-bandit
"UP",  # pyupgrade
"RUF100",  # unused noqa
```

as well as specify what is fixable:

```
fixable = ["C4", "E", "F", "I", "UP"]
```

Reply:
Well we're still overriding `select` so I don't see why we'd mix `select` and `select-extend`, but I've added your suggestions to the list. I wonder how opinionated we should be on things like this.

With my own cookiecutter, I go quite strongly opinionated because it is for my own use - I just do things how I like them.

With something like this it feels like we should be cover the basics without being too opinionated. Lots of people with different opinions generate packages from this cookiecutter, so it should encompass some best practices, but steer clear of being too rigid on more subjective things. If this repo was a food, it should be vanilla ice cream, not marmite on toast. Not necessarily anybody's favourite, but basically acceptable to everyone.

I might be rambling a bit here. I did the thing you asked for.